synapse/config/repository.py

```diff
@@ -202,6 +215,21 @@ def generate_config_section(self, data_dir_path, **kwargs):
 
         return (
             r"""
+        ##
+        ## Media Store
+        ##
+
+        # Is an external media repo in use?
```

Comment:
It seems very confusing to have two settings, rather than a single `enable_media_repo`. Can we not combine them?